analyzers/conversion_overflow.go

```diff
@@ -224,3 +291,71 @@ func isIntOverflow(src string, dst string) bool {
 
 	return false
 }
+
+func isRelevantBinOp(binOp *ssa.BinOp, x ssa.Value) bool {
+	return (binOp.X == x || binOp.Y == x) &&
+		(binOp.Op == token.LSS || binOp.Op == token.LEQ || binOp.Op == token.GTR || binOp.Op == token.GEQ)
+}
+
+func extractConst(binOp *ssa.BinOp) *ssa.Const {
```

Comment:
Maybe name this something like `constFromBoundCheck` to match other naming?

Reply:
resolved